content/en/docs/project-user-guide/application-workloads/routes.md

```diff
@@ -0,0 +1,156 @@
+
+   {{</ notice >}}
+
+   ![access-method-nodeport](/images/docs/project-user-guide/application-workloads/routes/access-method-nodeport.png)
+
+   ![access-method-loadbalancer](/images/docs/project-user-guide/application-workloads/routes/access-method-loadbalancer.png)
+
+## Create a Route
+
+### Step 1: Configure basic information
+
+1. Choose **Application Workloads** > **Routes** on the left navigation bar and click **Create** on the right.
+
+   ![create-route](/images/docs/project-user-guide/application-workloads/routes/create-route.png)
+
+2. On the **Basic Info** tab, configure the basic information about the Route and click **Next**.
+   * **Name**: Name of the Route, which is used as a unique identifier.
+   * **Alias**: Alias of the Route.
+   * **Description**: Description of the Route.
+
+   ![basic-info](/images/docs/project-user-guide/application-workloads/routes/basic-info.png)
+
+### Step 2: Configure Route rules
+
+1. On the **Route Rules** tab, click **Add Route Rule**.
+
+2. Select a mode, configure Route rules, click **√**, and click **Next**.
+
+   * **Auto Generate**: KubeSphere automatically generates a domain name in the `<Service name>.<Project name>.<Gateway address>.nip.io` format and the domain name is automatically resolved by [nip.io](https://nip.io/) into the gateway address. This mode supports only HTTP.
+     
+     * **Paths**: Map each service to a path. You can click **Add Path** to add multiple paths.
+     
+     ![auto-generate](/images/docs/project-user-guide/application-workloads/routes/auto-generate.png)
+     
+   * **Specify Domain**: A user-defined domain name is used. This mode supports both HTTP and HTTPS.
+     
+     * **HostName**: Set a domain name for the Route.
+     * **Protocol**: Select `http` or `https`. If `https` is selected, you need to select a Secret that contains the `tls.crt` (TLS certificate) and `tls.key` (TLS private key) keys used for encryption.
+     * **Paths**: Map each service to a path. You can click **Add Path** to add multiple paths.
+   
+     ![specify-domain](/images/docs/project-user-guide/application-workloads/routes/specify-domain.png)
+
+### (Optional) Step 3: Configure advanced settings
+
+1. On the **Advanced Settings** tab, select **Add Metadata**.
+
+2. Configure annotations and labels for the Route and click **Create**.
+
+   {{< notice note >}}
+
+   You can use annotations to customize the behavior of the Route. For more information, see the [official Nginx Ingress controller document](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/).
+
+   {{</ notice >}}
+
+   ![add-metadata](/images/docs/project-user-guide/application-workloads/routes/add-metadata.png)
+
+### Step 4: Obtain the domain name, service path, and gateway address
+
+1. Choose **Application Workloads** > **Routes** on the left navigation bar and click the name of the Route on the right.
```

Comment:
Choose **Application Workloads** > **Routes** on the left navigation bar
->
Choose **Routes** in **Application Workloads** on the left navigation bar